crossdock/client/errorstchout/behavior.go

```diff
@@ -150,6 +152,27 @@ func Run(t crossdock.T) {
 				assert.Equal(tchannel.ErrCodeUnexpected, code, "must produce transport error")
 			},
 		},
+		{
+			name:      "remote timeout error",
+			procedure: "waitfortimeout/raw",
+			body:      []byte("{}"),
+			headers:   []byte{0x0, 0x0},
+			encoding:  "raw",
+			validate: func(res3 []byte, isAppErr bool, err error) {
+				assert.Error(err, "is error")
```

Comment:
nit: Maybe use "request must fail"

Also, if this returns false, there's no point in doing any of the other checks so return right away:

```go
if !assert.Error(err, "...") {
    return
}
```